test-apps/ui-test-app/src/frontend/appui/frontstages/Settings.tsx

```diff
@@ -52,3 +54,18 @@ function SettingsItem(props: SettingsItemProps) {
     </div>
   );
 }
+
+export function TargetSettingsPageComponent() {
```

Comment:
Could you add this to the providers for the new appui-test-providers package?

Reply:
Enabled v2 targets in appui-test-app.